tests/test_evaluator_discovery.py

```
@@ -594,6 +594,206 @@ def test_parse_timeout_exceeds_max(self, tmp_path, caplog):
+        yml.write_text(
+            """
+name: test-eval
+description: Test evaluator
+model_requirement:
+  tier: opus
+prompt: "Test prompt"
+output_suffix: TEST
+"""
+        )
+
+        with pytest.raises(EvaluatorParseError, match="model_requirement.family"):
```

Ruff (0.14.14)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)